domains/kevin-xavier.json

```diff
@@ -0,0 +1,10 @@
+{
+  "subdomain": "kevin-xavier.is-a.dev",
+  "owner": {
+    "email": "iamkevinxavi@gmail.com",
+    "github": "keviiixaviiii"
+  },
+  "records": {
+    "CNAME": "keviiixaviiii.github.io/My-Portfolio/"

```

Comment:
CNAME can not contain `/`

Reply:
Verify your is-a.dev Domain with GitHub Pages

Reply:
created new Json file

Reply:
recently updated file